activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -280,7 +280,10 @@ def test_migrate_receives_correct_env_vars
       ENV['VERBOSE'] = 'false'
       ENV['VERSION'] = '4'
 
-      ActiveRecord::Migrator.expects(:migrate).with(ActiveRecord::Migrator.migrations_paths, 4)
+      ActiveRecord::Tasks::DatabaseTasks.expects(:migrations_paths).
```

Comment:
good call, done. I only couldn't use `original_migrations_path = ActiveRecord::Tasks::DatabaseTasks.migrations_paths` as this pulls in `Rails` dependency: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/tasks/database_tasks.rb#L59. But in any case `ActiveRecord::Tasks::DatabaseTasks.migrations_paths = nil` restores the previous 'before-test-run' behavior